### PR TITLE
regenerate ansible, katello and rex specs

### PR DESCRIPTION
**Pull request by evgeni**

### CODE REVIEW

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 1]
```diff
@@ -206,7 +189,13 @@ done
 %{gem_instdir}/package.json
 %{gem_instdir}/webpack
 
+%posttrans
+%{foreman_plugin_log}
```
Comment (evgeni) [inline packages/katello/rubygem-katello/rubygem-katello.spec:193]: Funny that we forgot to add that one here -.-

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 2]
```diff
@@ -1,28 +1,24 @@
...
+%global release 2
 
+Name: rubygem-%{gem_name}
 Version: %{mainver}
 Release: %{?prerelease:0.}%{release}%{?prerelease}%{?nightly}%{?dist}
-Group:   Applications/Systems
-License: GPLv2
-URL:     https://theforeman.org/plugins/katello
+Summary: Content and Subscription Management plugin for Foreman
+License: GPL-2.0
```
Comment (ekohl) [inline packages/katello/rubygem-katello/rubygem-katello.spec:15]: Isn't `GPLv2` the correct Fedora tag? I'm not sure we can use SPDX codes in EL8 (though Fedora has adopted them).

File: packages/katello/rubygem-katello/rubygem-katello.spec [hunk 3]
```diff
@@ -52,8 +48,6 @@ BuildRequires: rubygem(gettext_i18n_rails)
 BuildRequires: rubygem(apipie-rails) >= 0.5.14
 BuildRequires: rubygem(fx) < 1.0
 BuildRequires: rubygem(pg)
-BuildRequires: rubygem(runcible) >= 2.13.0
```
Comment (ekohl) [inline packages/katello/rubygem-katello/rubygem-katello.spec:55]: We can finally drop this, let's not forget the follow up PR to remove it.

File: packages/katello/rubygem-katello/rubygem-katello.spec (hunk 2 above)
Comment (evgeni) [reply packages/katello/rubygem-katello/rubygem-katello.spec:15]: maybe. https://github.com/theforeman/foreman-packaging/blob/rpm/develop/licenses.json doesn't list the "no later" version, so probably that's why it got reset to the value in https://github.com/Katello/katello/blob/2cad75d2d309afe0061f2f656d80409feff6c180/katello.gemspec#L10

File: packages/katello/rubygem-katello/rubygem-katello.spec (hunk 3 above)
Comment (evgeni) [reply packages/katello/rubygem-katello/rubygem-katello.spec:55]: https://github.com/theforeman/foreman-packaging/pull/9321

**Review by ekohl — APPROVED** [review]

**Comment by evgeni** [comment]
repoclosure is confused because the packages are in different repos actually, ignoring

**Pull request merged**